src/libraries/System.Diagnostics.Process/tests/ProcessStreamReadTests.cs

```diff
@@ -344,42 +344,6 @@ async private Task<bool> WaitPipeSignal(PipeStream pipe, int millisecond)
             }
         }
 
-        [PlatformSpecific(~TestPlatforms.Windows)] // currently on Windows these operations async-over-sync on Windows
-        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
-        public async Task ReadAsync_OutputStreams_Cancel_RespondsQuickly()
```

Comment:
This test should not be deleted.  Just replace the PlatformSpecific attribute with an ActiveIssue attribute.

Reply:
Addressed.